recipes/spdlog/all/conanfile.py

```diff
@@ -110,9 +142,16 @@ def generate(self):
     def _disable_werror(self):
         replace_in_file(self, os.path.join(self.source_folder, "cmake", "utils.cmake"), "/WX", "")
 
+    def _use_cpp20_for_std_format(self):
+        # This is properly set in later versions
```

Comment:
```suggestion
        # INFO: std_fmt rquires C++20: https://github.com/gabime/spdlog/blob/v1.12.0/CMakeLists.txt#L29
```
Let's try to keep the upstream reference to make it clear.